### PR TITLE
Update connector name to SAP On-premise

### PR DESCRIPTION
**Pull request by zweedao**

### CODE REVIEW

**Comment by zweedao** [comment]
@graceliching can you help to review & merge this :) Some modification after the SAP connector review on Fri.

File: connectors/sap.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ Download the SAP JCO connector libraries from the link below. Please ensure that
 
 https://support.sap.com/en/product/connectors/jco.html
 
-Once the libraries are downloaded, place them in the `/lib/` directory of the OPA.
+Once the downloaded libraries, place them in the `/lib/` directory of the OPA.
```
Comment (graceliching) [inline connectors/sap.md]: The old phrasing is fine here, the new one is grammatically incorrect :)

File: connectors/sap/action-send-idoc.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ date: 2018-12-06 06:00:00 Z
 
 ![Action Send IDoc](/assets/images/connectors/sap/action-send-idoc.png)
 
-This action allows you to send IDoc to your SAP system. Note that you must [configure IDocs in your SAP system](https://docs.workato.com/connectors/sap.html#configure-idoc-in-sap-to-work-with-workato) before using this action.
+This action allows you to send IDoc to on-premise SAP system, or to SAP system on a server behind the corporate firewall. Note that you must [configure IDocs in your SAP system](https://docs.workato.com/connectors/sap.html#configure-idoc-in-sap-to-work-with-workato) before using this action.
```
Comment (graceliching) [inline connectors/sap/action-send-idoc.md]: This action allows you to send IDocs to an on-prem SAP system, or to an SAP system on a server behind the corporate firewall. Note that you must [configure IDocs in your SAP system](https://docs.workato.com/connectors/sap.html#configure-idoc-in-sap-to-work-with-workato) before using this action.

File: connectors/sap/trigger-new-idoc.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ date: 2018-12-06 06:00:00 Z
 
 ![Trigger New IDoc](/assets/images/connectors/sap/trigger-new-idoc.png)
 
-This trigger allows you to listen to and receive IDocs from SAP. Note that you must [configure IDocs in your SAP system](https://docs.workato.com/connectors/sap.html#configure-idoc-in-sap-to-work-with-workato) before using this trigger.
+This trigger allows you to listen to and receive IDocs from on-premise SAP system, or from SAP system on a server behind the corporate firewall. Note that you must [configure IDocs in your SAP system](https://docs.workato.com/connectors/sap.html#configure-idoc-in-sap-to-work-with-workato) before using this trigger.
```
Comment (graceliching) [inline connectors/sap/trigger-new-idoc.md]: This trigger allows you to listen to and receive IDocs from an on-prem SAP system, or from an SAP system on a server behind the corporate firewall. Note that you must [configure IDocs in your SAP system](https://docs.workato.com/connectors/sap.html#configure-idoc-in-sap-to-work-with-workato) before using this trigger.

File: connectors/sap.md (hunk 1 above)
Comment (zweedao) [reply connectors/sap.md]: yeah hmm why did I change it, will change back

**Comment by zweedao** [comment]
@graceliching can you check again. Something wrong with github. The version here was not the latest version that I pushed last week.

**Pull request merged**